data/AzraNaseri.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Azra Naseri",
+  "bio": "Frontend developer | JavaScript | react |Java | open source contributor",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/AzraNaseri",
+      "icon": "FaGithub"
+      
+    }
+  ],
+  "social": [
+    {
+      "name": "Follow me on LinkedIn",
```

Socials don't require a `name` key. Also there is a missing `s` in `"socials"`

```suggestion
  "socials": [
    {
```

---

I updated the codes and implemented what you mentioned. So why has it still not merged?

---

and how can i open my codes in the browser i mean to see the result (profile)?

---

You'll find your profile at https://biodrop.io/AzraNaseri after this PR is being merged and deployed!